test/danetest.c

```diff
@@ -168,7 +166,7 @@ static char *read_to_eol(BIO *f)
     }
 
     /* Trim trailing whitespace */
-    while (n > 0 && isspace(_UC(buf[n - 1])))
+    while (n > 0 && isspace(buf[n - 1]))
```

Comment:
It's inappropriate to just remove _UC here. If you want to remove you have to declare buf as unsigned char.